src/controllers/IllPersonController.ts

```diff
@@ -62,30 +61,25 @@ export class IllPersonController extends BaseController<FormBody> {
     }
 
     protected prepareSessionModelPriorSave(appeal: Appeal, value: FormBody): Appeal {
-
+        // TBD: Remove OtherReason object when creating Illness if not Multiple reasons
```

Comment:
Is this comment still relevant

Reply:
Thanks Lewis, I think is important as a note for the team when the Illness journey object starts and how to behave.